sqs/src/test/java/akka/stream/alpakka/sqs/javadsl/BaseSqsTest.java

```diff
@@ -44,13 +45,17 @@ public static void setup() {
 
   @AfterClass
   public static void teardown() {
+    if (sqsClientForClose != null) {
```

Comment:
I don't understand the context of why this is necessary.  How come we can't just always call `sysClient.close`?  What is the purpose of `sqsClientForClose`?

Reply:
This is a "static vs. instance" dance.

Reply:
Ah, I didn't notice that it was static.